File: Java/프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.java
import java.util.*;

class Solution {
    public int[] solution(String[] wallpaper) {
        int minY = 52;
        int minX = 52;
        int maxY = 0;
        int maxX = 0;
        
        for (int y = 0; y<wallpaper.length; y++) {
            for (int x = 0; x<wallpaper[y].length(); x++) {
                char item = wallpaper[y].charAt(x);
                if (item == '#') {
                    minY = Math.min(minY, y);
                    minX = Math.min(minX, x);
                    maxY = Math.max(maxY, y+1);
                    maxX = Math.max(maxX, x+1);
                }
            }
        }
        int[] answer = {minY, minX, maxY, maxX};
        return answer;
    }
}

File: Swift/프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.swift
import Foundation

func solution(_ wallpaper:[String]) -> [Int] {
    var minY = Int.max
    var minX = Int.max
    var maxY = Int.min
    var maxX = Int.min
    for y in 0..<wallpaper.count {
        let rowString = wallpaper[y]
        for x in 0..<rowString.count {
            let index = rowString.index(rowString.startIndex, offsetBy: x)
            let char = String(rowString[index])
            if char == "#" {
                minY = min(minY, y)
                minX = min(minX, x)
                maxY = max(maxY, y+1)
                maxX = max(maxX, x+1)
            }
        }
    }
    return [minY, minX, maxY, maxX]
}